src/test/java/com/philips/research/spdxbuilder/persistence/ort/OrtReaderTest.java

```diff
@@ -13,21 +13,43 @@
 import java.nio.file.Path;
 import java.util.List;
 
+import static junit.framework.TestCase.assertTrue;
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 
 
 class OrtReaderTest {
-    private static final File ORT_SAMPLE = Path.of("src", "test", "resources", "ort_sample.yml").toFile();
 
-    private final BillOfMaterials bom = new BillOfMaterials();
-
-    @Test
-    void loadsOrtSample() {
-        new OrtReader(ORT_SAMPLE)
-                .defineProjectPackage("NPM::mime-types:2.1.18", URI.create("pkg:npm/mime-types@2.1.18"))
-                .excludeScopes("NPM::mime-types:2.1.18", List.of("test*"))
-                .read(bom);
-
-        assertThat(bom.getPackages()).hasSize(1 + 2);
-    }
+  private static final File ORT_SAMPLE1 = getPathOfFile("ort_sample1.yml");
+  private static final File ORT_SAMPLE2 = getPathOfFile("ort_sample2.yml");
```

Comment:
My preference (because a constant is "simpler code" than a function) would be to make the path of the sample files a constant, like e.g.:
```
private static final Path SAMPLES_DIR = Path.of("src", "test", "resources");
private static final File ORT_SAMPLE = SAMPLES_DIR.resolve("ort_sample.yaml").toFile();
private static final File ORT_SAMPLE = SAMPLES_DIR.resolve("ort_with_issue.yaml").toFile();
```